worlds/animal_well/region_data.py

```diff
@@ -490,13 +492,15 @@ class AWData(NamedTuple):
             AWData(AWType.location, loc_type=LocType.fruit),
         rname.bear_ladder_after_chameleon:
             AWData(AWType.region),  # just press a button
-        # just throw the ball into the hole, it'll hit the yellow button like 99% of the time
+        rname.bear_slink_room:
+            AWData(AWType.region),  # just go up
```

Comment:
```suggestion
        rname.bear_slink_room:
            AWData(AWType.region, [[iname.bubble], [iname.disc_hop], [iname.wheel_hop]]),  # just go up
```
There's way too much vertical distance needed to just jump up this -- you need a bubble, a kinda tricky disc hop, or a wheel hop

Reply:
I think this is up there for "the most wrong I've been about something that doesn't matter".

Reply:
idk I think that goes to the matchbox up above lmao